src/CMakeLists.txt

```diff
@@ -643,6 +643,11 @@ if (NUPIC_BUILD_PYEXT_MODULES)
       -DSWIG_PYTHON_LEGACY_BOOL
       -I${SWIG_DIR}/python
       -I${SWIG_DIR}
+   # fix for swig python.swg not found on MinGW
+   # https://github.com/numenta/nupic.core/pull/1321#issuecomment-356665619
+   -python 
+   -I${SWIG_PATH}/Lib 
+   -I${SWIG_PATH}/Lib/python
```

Comment:
This commit moves us from 
> https://github.com/numenta/nupic.core/pull/1321#issuecomment-356665619
to 
> https://ci.appveyor.com/project/numenta-ci/nupic-core/build/0.3.0.2227/job/hcx4ji7p9s5pbg0i#L1357 

@rhyolight do you know someone on Windows who can look into the swig paths? Where the new swig binary is located and what paths it expects

Reply:
@breznak I added windows support to your branch with this PR breznak/nupic.core#3
Here is the windows build log: 
- https://ci.appveyor.com/project/lscheinkman/nupic-core/build/0.3.0.3/job/fq7cop9ipp4929bf

CC: @rhyolight